docs/en/04_Changelogs/4.11.0.md

```diff
@@ -19,102 +20,104 @@ title: 4.11.0 (unreleased)
+- You will need to deploy the generated schema to your test and production environments. There are several ways to do this depending on your hosting situation – see the [deploying the schema](/developer_guides/graphql/getting_started/deploying_the_schema) documentation for details.
 
-If you were already using GraphQL v3 for your own custom schema and queries and want to upgrade to v4, you will also need to read the [Upgrading to GraphQL 4](/upgrading/upgrading_to_graphql_4) documentation, and are encouraged to read the [GraphQL documentation](/developer_guides/graphql/) generally to make sure your existing knowledge carries over to the new major release.
+If you were already using `silverstripe/graphql` v3 for your own custom schema and queries and want to upgrade to v4, you will also need to read the [Upgrading to GraphQL 4](/upgrading/upgrading_to_graphql_4) documentation, and are encouraged to read the [GraphQL documentation](/developer_guides/graphql/) in general to make sure your existing knowledge carries over to the new major release.
 
-### That sounds risky, do I absolutely have to use version 4?
+### That sounds risky, do I absolutely have to use version 4? {#graphqlv4-can-i-use-v3}
 
 Silverstripe CMS has been shipping with dual support for `silverstripe/graphql` v3 and v4 since the 4.8 release. Until now `silverstripe/graphql` v4 had been in alpha and you had to explicitly opt-in to get it. At Silverstripe, we are already using `silverstripe/graphql` v4 in production on several projects.
 
 All the supported Silverstripe CMS modules that use `silverstripe/graphql` have dual-support. If you wish to stay on `silverstripe/graphql` v3, you can do so and it will not block you from upgrading to Silverstripe CMS 4.11.
 
-#### Opting out of `silverstripe/graphql` version 4 and sticking to version 3
+#### Opting out of `silverstripe/graphql` version 4 and sticking to version 3 {#graphqlv4-how-to-use-v3}
 
-If your project composer.json file already explicitly requires silverstripe/graphql, you don’t need to do anything.
+If your project composer.json file already explicitly requires `silverstripe/graphql` version 3, you don’t need to do anything.
 
-If your project uses silverstripe/recipe-cms, it will install silverstripe/graphql:^4.0 when you upgrade to the 4.11 release. To stay on silverstripe/graphql:^3, you'll need to "inline" the `silverstripe/recipe-cms` requirements in your root composer.json and change `silverstripe/graphql` to `^3`.
+If your project uses `silverstripe/recipe-cms`, it will install `silverstripe/graphql:^4.0` when you upgrade to the 4.11.0 release. To stay on `silverstripe/graphql:^3`, you'll need to ["inline"](https://github.com/silverstripe/recipe-plugin#inlining-recipes) the `silverstripe/recipe-cms` dependencies into your root composer.json and change the constraint for `silverstripe/graphql` to `^3`.
```

Comment:
Inlining shouldn't actually be necessary - recipe-cms has dual support, so just updating the project's composer.json to explicitly require graphql v3 (as per the `composer require` command below) should be sufficient.